### PR TITLE
json-schema metadata

### PR DESCRIPTION
**Pull request by shimks**
Docs for https://github.com/strongloop/loopback-next/issues/786

### CODE REVIEW

**Review by b-admike — APPROVED** [review]
👍

File: pages/en/lb4/Schemas.md [hunk 1]
```diff
@@ -144,6 +144,76 @@ class Customer extends Entity {
 }
 ```
 
+### JSON Schema inferrence
```
Comment (bschrammIBM) [inline pages/en/lb4/Schemas.md]: (sp) inference

File: pages/en/lb4/Schemas.md [hunk 2]
```diff
@@ -144,6 +144,76 @@ class Customer extends Entity {
...
+Following are the supported keywords that can be explicitly passed into the decorators
+to better tailor towards the JSON Schema being produced.
+
+| Keywords    | Decorator   | Type    | Default      | Description                                             |
+|-------------|-------------|---------|--------------|---------------------------------------------------------|
+| title       | `@model`    | string  | *model name* | Name of the model                                       |
+| description | `@model`    | string  |              | Description of the model                                |
+| array       | `@property` | boolean |              | Used to specify whether the property is an array or not |
+| required    | `@property` | boolean |              | Used to specify whether the property is required or not |
+
```
Comment (kjdelisle) [inline pages/en/lb4/Schemas.md:223]: I would add one of those decoration tags we have to this section to say that it's a work-in-progress. There are most likely other JSON schema keywords/features that we'll want to support in the future, so letting people know that it's not yet feature complete in that respect would be helpful.

File: pages/en/lb4/Schemas.md [hunk 3]
```diff
@@ -144,6 +144,76 @@ class Customer extends Entity {
 }
 ```
 
+### JSON Schema inferrence
+With type information inferred from using `@model` and `@property` decorators,
+we've created a module to easily build a JSON Schema from a decorated model.
+In the `@loopback/repository-json-schema` module, we provide `getJsonSchema`
+which accesses the metadata stored by the decorators to build a matching
+JSON Schema of your model.
+
```
Comment (bschrammIBM) [inline pages/en/lb4/Schemas.md]: Suggested rewording: 
Use the `@loopback/repository-json-schema` module to build a JSON schema from a decorated model. Type information is inferred from the `@model` and `@property` decorators. The `@loopback/repository-json-schema` module contains the `getJsonSchema` function to access the metadata stored by the decorators to build a matching JSON Schema of your model.

File: pages/en/lb4/Schemas.md (hunk 2 above)
Comment (bschrammIBM) [reply pages/en/lb4/Schemas.md:223]: I added edits in-line.

File: pages/en/lb4/Schemas.md [hunk 4]
```diff
@@ -144,6 +144,85 @@ class Customer extends Entity {
 }
 ```
 
+### JSON Schema inference
+With type information inferred from using `@model` and `@property` decorators,
+we've created a module to easily build a JSON Schema from a decorated model.
```
Comment (bschrammIBM) [inline pages/en/lb4/Schemas.md]: Delete the first sentence:
+With type information inferred from using `@model` and `@property` decorators,
 +we've created a module to easily build a JSON Schema from a decorated model.

File: pages/en/lb4/Schemas.md (hunk 2 above)
Comment (bschrammIBM) [reply pages/en/lb4/Schemas.md:223]: One additional edit needed.

**Review by bschrammIBM — APPROVED** [review]

**Pull request merged**